### PR TITLE
Contrast 11616 api credentials

### PR DESCRIPTION
**Pull request by kaitlynmortimer**
Navigation updates: https://bitbucket.org/contrastsecurity/open-docs/pull-requests/128/contrast-11616-api-credentials/diff

### CODE REVIEW

File: content/admin/org_settings/API-credentials.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+tags: "Admin system settings security API keys configuration"
+-->
+
+When you download the Contrast JVM plugin (also called the Engine), it comes pre-fitted with a set of randomly generated credentials for your organization that don't involve the passwords of any users in the organization. When the plugin communicates with the Contrast application, it authenticates using these credentials. 
+ 
+<a href="assets/images/API-credentials-1.png" rel="lightbox" title="Default authentication"><img class="thumbnail" src="assets/images/API-credentials-1.png"/></a>
+
+## How It Works 
+
+Contrast added another layer of security through an organization API key that you can enable and manage on your own. In the case of a security breach, an unauthorized user can't submit forged or malicious data to your organization because their organization API key is wrong. With the API key enabled, authentication follows the process shown in the image below.
```
Comment (cgtwin) [inline content/admin/org_settings/API-credentials.md]: API Key is always enabled, and there is not way to turn it off. 

Probably you could remove "you can enable" and "With the API key enabled"

File: content/admin/org_settings/API-credentials.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!--
+title: "Security API Keys"
+description: "About article for the API Keys"
+tags: "Admin system settings security API keys configuration"
+-->
+
+When you download the Contrast JVM plugin (also called the Engine), it comes pre-fitted with a set of randomly generated credentials for your organization that don't involve the passwords of any users in the organization. When the plugin communicates with the Contrast application, it authenticates using these credentials.
```
Comment (cgtwin) [inline content/admin/org_settings/API-credentials.md]: Not sure if we still want to call it "Engine". In the TeamServer UI, we changed all the labels/texts from Engine to Agent

File: content/admin/sys_settings/SysSettingsGlance.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Get a glimpse into license allocation, see if any licenses are nearing expiratio
 Manage the library compliance policy for your organization(s). Setup restrictions or version requirements for library usage which will flag violations that break compliance settings.
 
 ## Security
-Manage [password policies](admin-systemsettings.html#pwd), [two-step verification](admin-orgsecurity.html#security-tsv), and [API and application key management](admin-orgsecurity.html#key-manage). Any change to this setting will require a system restart. 
+Manage [password policies](admin-systemsecurity.html#pwd), [two-step verification](admin-orgsecurity.html#security-tsv), and [API and application key management](admin-systemsecurity.html#manage-api). Any change to this setting will require a system restart.
```
Comment (cgtwin) [inline content/admin/sys_settings/SysSettingsGlance.md]: IMO system restart sounds like computer restart/reboot

Comment (kaitlynmortimer) [reply content/admin/sys_settings/SysSettingsGlance.md]: @cgtwin As opposed to the agents being reconfigured?

File: content/admin/org_settings/API-credentials.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<!--
+title: "Security API Keys"
+description: "About article for the API Keys"
+tags: "Admin system settings security API keys configuration"
+-->
+
+When you download the Contrast JVM plugin (also called "the agent"), it comes pre-fitted with a set of randomly generated credentials for your organization that don't involve the passwords of any users in the organization. When the plugin communicates with the Contrast application, it authenticates using these credentials. 
+ 
+<a href="assets/images/API-credentials-1.png" rel="lightbox" title="Default authentication"><img class="thumbnail" src="assets/images/API-credentials-1.png"/></a>
```
[the remark this answers is not in the document]
Comment (kaitlynmortimer) [reply content/admin/org_settings/API-credentials.md]: @serchduran I think you're right - I got 1 and 2 mixed up when I named them. Thanks for catching this!

**Pull request merged**